src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        Optional<Integer> result = numbers.stream()
+                .map(n -> n.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
+                .filter(n -> n % 2 == 0)
+                .min(Comparator.comparing(Integer::valueOf));
```

Comment:
```suggestion
                .min();
```
You do not need to add comparator here

Reply:
But it requires argument, i can simplify it to : min(Integer::compare), it will be ok?